vulnerable_people_form/form_pages/shared/location_tier.py

```diff
@@ -30,12 +30,15 @@ def update_location_tier_by_uprn(uprn, app):
 
         lad_code = get_ladcode_from_uprn(uprn)
         if app.shielding_advice.is_la_shielding(lad_code):
-            set_shielding_advice(1)
+            set_shielding_advice(ShieldingAdvice.ADVISED_TO_SHIELD.value)
```

Comment:
This method seems to do more than just update location tier it sets the shielding advice? Should this be broken down into a few methods:

one to set the tier
one to set the shielding advice
one to set the override for the postcode

?

Reply:
I've changed the name of the function to represent what it does better and it now calls the different functions you suggested,